docs/releases/slashing-release/interfaces/slashing-parameter-configuration.md

```diff
@@ -10,28 +10,33 @@ interface IAVSDirectory {
      * @notice this struct is used in queueAllocation, queueDeallocation in order to specify
      * an operator's slashability for a certain operator set
      *
-     * @param operatorSet the operator set to change slashing parameters for
+     * @param strategy strategy to adjust allocations/deallocations for
+     * @param operatorSets the operator sets to adjust magnitudes for
      * @param magnitudeDiff magnitude difference; the difference in proportional parts of the operator's slashable stake
      * that the operator set is getting. This struct is used either in allocating or deallocating
      * slashable stake to an operator set. Slashable stake for an operator set is
      * (slashableMagnitude / sum of all slashableMagnitudes for the strategy/operator + nonSlashableMagnitude) of
      * an operator's delegated stake.
      */
     struct MagnitudeAdjustment {
-        OperatorSet operatorSet;
-        uint64 magnitudeDiff;
+        IStrategy strategy;
+        OperatorSet[] operatorSets;
+        uint64[] magnitudeDiffs;
     }
 
     /**
-     * @notice struct used for queued deallocations. Hash of struct is set to true in a mapping
+     * @notice struct used for queued deallocations. Hash of struct is set in storage to be referenced later
+     * when completing deallocations.
+     * @param operator address for performing deallocations for
+     * @param nonce
      */
     struct QueuedDeallocation {
         address operator;
         uint16 nonce;
-        IStrategy strategy;
-        OperatorSet operatorSet;
-        uint64 deallocationAmount;
         uint64 queuedTotalMagnitude;
```

Comment:
not exactly sure what this is

Reply:
Ah natspec was incomplete. The totalMagnitude at queue time so that upon completion, we can use the totalMag at queue time and totalMag at completion to calculate the actual amount that is being added back to nonslashable